backend-cf/.env.example

```
@@ -0,0 +1,37 @@
+# Application URLs (Required)
+FRONTEND_URL=https://your-app-domain.com
+EXTENSION_URL=chrome-extension://your-extension-id
+
+# Stripe Payment Processing (Optional)
+# Get these from https://dashboard.stripe.com
+STRIPE_SECRET_KEY=sk_live_your_stripe_secret_key
+STRIPE_WEBHOOK_SECRET=whsec_your_stripe_webhook_secret
+
+# AI Provider Keys (Required)
+# CometAPI is used as the primary provider to avoid regional restrictions
+COMET_API_KEY=your_comet_api_key  # Required for proxy functionality
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)